src/main

```diff
@@ -18,6 +18,8 @@ if ! git rev-parse --git-dir &>/dev/null; then
 fi
 
 directory=$(git worktree list | awk '{print $1}' | xargs -n1 basename | fzf --print-query --prompt="Select or create worktree: " | tail -1)
+# Replace all spaces with a '-' to enable quicker directory naming
+directory=$(echo $directory | tr ' ' '-')
```

Comment:
What happens with trailing spaces? Is it too opinionated to remove trailing spaces? It's a common pattern to remove trailing spaces in code ..